### PR TITLE
Tests, set Timeout and last details

### PR DESCRIPTION
**Pull request by manuelperdigons**
Hi br, here I changed all little details we spoke before, if you can verify and confirm would be awesome, also set timeout in axios function of 1sec as coding challenge specified and finally added a test just to practice with mocha and chai a bit, tell me if you see something else or any other idea you could have

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -1,3 +1,14 @@
...
+const { assert } = require('chai');
+const flightController = require('../controllers/index');
+const flightsAPI = require('../controllers/getFlightsData');
 
-describe()
\ No newline at end of file
+describe('controllers/index.js', function () {
+
+    it('should remove duplicated flights', async function () {
+        const source1 = await flightsAPI.getFlights(process.env.LINK_ONE);
```
Comment (brahyam) [inline test/test.js]: As you are not really sure what the API is gonna return (maybe there could be repeated flights, maybe not, maybe the API could fail to response at all) its better to test with local sample data. You could create a file with some sample flights list (usually called fixtures). eg. with 2 lists that have flights (s1, s2) in common and another one that should be the expected result (r1). and them declare a test where you:
`assert.isEqual(await flightController.mergeFlightsLists(s1,s2),r1)`
We could have a call to check this together

File: test/test.js [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+        const source2 = await flightsAPI.getFlights(process.env.LINK_TWO);
+        assert.isArray(await flightController.mergeFlightLists(source1, source2), 'Create final flights list correctly');
```
Comment (brahyam) [inline test/test.js]: Here you are asserting only that the result is an Array but you still need to test if there are any duplicates.

**Comment by manuelperdigons** [comment]
Done brrrro, I requested a review from you just to get all clear from both. Thanks again for your help.

**Comment by manuelperdigons** [comment]
Done brrrro, I requested a review from you just to get all clear from both. Thanks again for your help.

**Review by brahyam — APPROVED** [review]
All good stuff, great job!
